src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,54 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  static addAnimal(animal) {
+    this.alive.push(animal);
+  }
+
+  static removeAnimal(animal) {
+    const indexOfAnimal = this.alive.indexOf(animal);
+
+    if (indexOfAnimal !== -1) {
+      this.alive.splice(indexOfAnimal, 1);
+    }
```

Comment:
[CODE PERFOMANCE] Try to avoid nesting loops (indexOf + splice), use filter instead